packages/template-block/src/TemplateBlock.tsx

```diff
@@ -0,0 +1,218 @@
+                        padding: isCardPaddingCustom
+                            ? `${cardPaddingCustomTop} ${cardPaddingCustomRight} ${cardPaddingCustomBottom} ${cardPaddingCustomLeft}`
+                            : cardPaddingValues[cardPaddingSimple],
+                    }}
+                >
+                    {isEditing && lastErrorMessage !== '' && <AlertError errorMessage={lastErrorMessage} />}
+                    <div
+                        className="tw-flex"
+                        style={{
+                            flexDirection,
+                            gap: GAP,
+                            alignItems: isRows ? textAnchoringHorizontal : textAnchoringVertical,
+                        }}
+                    >
+                        {hasPreview && (
+                            <TemplatePreview
+                                appBridge={appBridge}
+                                template={selectedTemplate}
+                                onUpdateTemplate={updateTemplateIdsFromKey}
+                                onUpdateTemplateTitle={updateTemplateTitle}
+                                onUpdateTemplateDescription={updateTemplateDescription}
+                                onError={setLastErrorMessage}
+                            />
+                        )}
+                        <div
+                            className={merge(['tw-flex', isRows && hasPreview ? 'tw-flex-col' : 'tw-flex-row'])}
+                            style={{
+                                width: isRows && hasPreview ? `${textRatio}%` : '100%',
+                                gap: GAP,
+                            }}
+                        >
+                            <div className="tw-grow tw-min-w-0">
+                                <div className="tw-mb-2">
+                                    {isEditing ? (
+                                        <span data-test-id="template-block-title-edit">
+                                            <TextInput
+                                                id={`${blockId}-title`}
+                                                value={templateTitle}
+                                                placeholder={isEditing ? 'Template Name' : undefined}
+                                                onChange={updateTemplateTitle}
+                                            />
+                                        </span>
+                                    ) : (
+                                        <Heading size="xx-large" weight="strong">
+                                            <span data-test-id="template-block-title">{templateTitle}</span>
+                                        </Heading>
+                                    )}
+                                    <div>
+                                        <Text size="small">
+                                            <span data-test-id="template-block-page-count">
```

Comment:
Color of page count and description should be text-weak.

Reply:
Fixed